pkg_resources/__init__.py

```diff
@@ -102,7 +103,7 @@
 from platformdirs import user_cache_dir as _user_cache_dir
 
 if TYPE_CHECKING:
-    from _typeshed import BytesPath, StrPath, StrOrBytesPath
+    from _typeshed import StrPath, BytesPath, StrOrBytesPath
```

Comment:
Unsure why this order has changed. Maybe something was removed and re-added. At some point I want to introduce `ruff check --select I`, which suggests:

```suggestion
    from _typeshed import BytesPath, StrOrBytesPath, StrPath
```

Reply:
Probably removed/added, merge+autofix, or something like that. I have prepped #4392 for automatic import sorting.